doc/assessment.md

```diff
@@ -19,36 +19,65 @@ Each assessment is a single directory in the `assessments` folder. The directory
 }
 ```
 
-* [Format specification for assessment `infoAssessment.json`](https://github.com/PrairieLearn/PrairieLearn/blob/master/schemas/schemas/infoAssessment.json)
+[Format specification for assessment `infoAssessment.json`](https://github.com/PrairieLearn/PrairieLearn/blob/master/schemas/schemas/infoAssessment.json)
```

Comment:
```suggestion
For more details, please see the [format specification for assessment `infoAssessment.json`](https://github.com/PrairieLearn/PrairieLearn/blob/master/schemas/schemas/infoAssessment.json).
```

Perhaps this should also go after the table of property values given below?

Reply:
done